### PR TITLE
[Chore] Bump version to 0.4.0

### PR DESCRIPTION
**Pull request by Thieurom**
## What happened 👀
Bump version to `0.4.0`

### CODE REVIEW

File: CryptoPrices.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -7,9 +7,6 @@
 	objects = {
 
 /* Begin PBXBuildFile section */
-		1C02C659294B007A00ED7DFC /* Styleguide in Frameworks */ = {isa = PBXBuildFile; productRef = 1C02C658294B007A00ED7DFC /* Styleguide */; };
-		1C02C65B294B008A00ED7DFC /* UseCases in Frameworks */ = {isa = PBXBuildFile; productRef = 1C02C65A294B008A00ED7DFC /* UseCases */; };
-		1C02C65F294B02B300ED7DFC /* Repositories in Frameworks */ = {isa = PBXBuildFile; productRef = 1C02C65E294B02B300ED7DFC /* Repositories */; };
```
Comment (minhnimble) [inline CryptoPrices.xcodeproj/project.pbxproj:12]: Wait is this intentional 🤔

**Review by minhnimble — APPROVED** [review]
I have a small question, rest LGTM 🚀

File: CryptoPrices.xcodeproj/project.pbxproj (hunk 1 above)
Comment (nkhanh44) [reply CryptoPrices.xcodeproj/project.pbxproj:12]: the same concern to me 😄

**Review by nkhanh44 — APPROVED** [review]
🚀

**Pull request merged**